packages/web/integrations/axelar/types.ts

```diff
@@ -29,10 +29,15 @@ export type SourceChain =
   | "Ethereum"
   | "Ropsten Test Network"
   | "Avalanche"
+  | "Avalanche FUJI C-Chain"
   | "Fantom"
+  | "Fantom Testnet"
   | "Polygon"
+  | "Mumbai"
   | "Moonbeam"
-  | "Binance";
+  | "Moonbase Alpha TestNet"
+  | "Binance Smart Chain"
```

Comment:
Looks like we need to update `EthClientChainIds_AxelarChainIdsMap` below to be "Binance Smart Chain"